### DISCUSSION TITLE
The way to make upload knowledge file faster

### DISCUSSION THREAD

**Discussion by trongnichi** [langgenius/dify#3283] [General]
Hi all, can I know the way everybody set up Docker to make upload knowledge files faster?

I uploaded the file with lower than 1MB, but it extremely slow, about 30 minutes to 1 hour or more, just to waiting for the file to available.
This caused my performance to really decline, how can I config Docker or something to upload faster?

Thank you all.

**Comment by xMarlonNx** [comment]
I would like to know this myself. I'm also locally hosting DIFY and file uploads are crazy slow.

**Reply by massif-01** [reply]
+1
